### PR TITLE
Libraries and tools for SRT

### PR DESCRIPTION
**Pull request by qiankehan**
Maintainer: me / @\<github-user> (find it by checking history of the package Makefile)
Compile tested: (x86_64, KVM, OpenWrt 21.02.2)
Run tested: (x86_64, KVM, OpenWrt 21.02.2, tests done)

Description:
Add package srt-utils, libsrt, and srt-live-server. These would provide live stream servers for Secure Reliable Tranport(SRT)

### CODE REVIEW

File: multimedia/srt-live-server/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+define Package/srt-live-server/description
+  srt-live-server(SLS) is an open source live streaming server for low latency based on
+  Secure Reliable Transport(SRT). Normally, the latency of transport by SLS is less than
+  1 second in internet.
+endef
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		               CXX="$(TARGET_CXX)" \
```
Comment (feckert) [inline multimedia/srt-live-server/Makefile:36]: The indentation does not fit please use `tabs`

File: multimedia/srt-live-server/Makefile [hunk 2]
```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=srt-live-server
+PKG_VERSION:=1.4.8
+PKG_RELEASE:=1
+
+PKG_SOURCE_NAME:=$(PKG_NAME)
```
Comment (feckert) [inline multimedia/srt-live-server/Makefile:7]: see above `libs/srt/Makefile`

File: multimedia/srt-live-server/Makefile [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/Edward-Wu/$(PKG_NAME)/archive/refs/tags/V$(PKG_VERSION).tar.gz?
+PKG_HASH:=3526a65e1d581b9ec62627d733d62a90dc074ae830e9bd7abe3f3b0a152b2786
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_NAME)-$(PKG_VERSION)
```
Comment (feckert) [inline multimedia/srt-live-server/Makefile:12]: see above `libs/srt/Makefile`

**Review by feckert — CHANGES_REQUESTED** [review]
I do not know if this is necessary, but could you please add a lincense header to your make files?
```
#
# Copyright (C) 2022 Han Han
#
# This is free software, licensed under the GNU General Public License v3.
# See /LICENSE for more information.
#
```

**Comment by qiankehan** [comment]
@feckert Thanks for your review. I am not familiar with the makefile of OpenWRT packages. So I copy some formats from other packages while I do not quite understand them.

**Comment by BKPepe** [comment]
Done in https://github.com/openwrt/packages/pull/25069, closing!

**Pull request closed without merging**